doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -1513,7 +1513,7 @@ In a goal like the following::
   =============
   m < 5 + n
 
-The tactic ``abstract: abs n`` first generalizes the goal with respect ton
+The tactic :g:`abstract: abs n` first generalizes the goal with respect to :math:`n`
```

Comment:
Despite this being done somewhere else, I'd advise against using the `:math:` scope in such case. `:g:` is fine.